concent_api/core/message_handlers.py

```diff
@@ -805,12 +805,6 @@ def handle_send_force_payment(
     validate_ethereum_addresses(requestor_eth_address, provider_eth_address)
     requestor_ethereum_public_key = hex_to_bytes_convert(task_to_compute.requestor_ethereum_public_key)
 
-    validate_that_golem_messages_are_signed_with_key(
```

Comment:
Why are you removing this? This is not mentioned in issue.

Reply:
this validation was wrong and was causing newly added tests to fail. Ownership is already checked at the begining og the message handler:
```python
    try:
        for subtask_results_accepted in client_message.subtask_results_accepted_list:
            subtask_results_accepted.validate_ownership_chain(settings.CONCENT_PUBLIC_KEY)
    except InvalidSignature:
        return ServiceRefused(
            reason=ServiceRefused.REASON.InvalidRequest
        )
```

Reply:
Yeah.

BTW, It's great that it's in a separate commit so it's not mixed with other changes and pretty easy to review.

The commit description could be better though. `Remove wrong validation` does not say why it's wrong and this a crucial information when reviewing a change or even trying to describe it in a changelog.

Reply:
Just to clarify, what I have meant was rather "This is not mentioned in the issue, and you are removing existing validation, that look suspicious" ;)